scripts/training_loop.py

```diff
@@ -170,12 +348,15 @@ def main():
         device=DEVICE,
     )
 
-    file_name = f"RobotId{gripper_config.gripper_id}_EnvType{env_config.env_type}_ObsType{env_config.object_type}_Seed{learning_config.seed}_{str(agent)[40:43]}"
+    file_name = f"Friday_14_RobotId{gripper_config.gripper_id}_EnvType{env_config.env_type}_ObsType{env_config.object_type}_Seed{learning_config.seed}_{str(agent)[40:43]}"
```

Comment:
Extract this out a bit further - Friday_14_RobotID stuff shouldn't be so hard coded

Reply:
https://github.com/UoA-CARES/Gripper-Code/pull/12/commits/3f8a5405085e28572d62b198d18575137ba75f95